src/pages/studyView/StudyViewPageStore.ts

```diff
@@ -8938,12 +8905,9 @@ export class StudyViewPageStore
     // a row represents a list of patients that either have or have not recieved
     // a specific treatment
     public readonly sampleTreatments = remoteData({
-        await: () => [this.selectedSamples, this.displaySampleTreatments],
+        await: () => [this.displaySampleTreatments],
```

Comment:
i think displaySampleTreatments is a boolean?  maybe we should rename it to shouldDisplaySampleTreatments?

Reply:
and while we're at it shouldDisplayPatientTreatments?